example/android/settings.gradle

```diff
@@ -18,7 +18,7 @@ pluginManagement {
 
 plugins {
     id "dev.flutter.flutter-plugin-loader" version "1.0.0"
-    id "com.android.application" version "8.7.3" apply false
+    id "com.android.application" version "8.6.1" apply false
```

Comment:
is that change on purpose?

Reply:
@remonh87

Reply:
yes I couldn't compile the app on my android studio version but I can revert it otherwise.